File: API/Models/ShoppingCart.cs
namespace API.Models
{
    public class ShoppingCart
    {
        public int Id { get; set; }
        public int  ProductId { get; set; }
        public string ProductName { get; set; }
        public double ProductPric { get; set; }
        public string SelectedSize { get; set; }
        public int Quantity { get; set; }
        public string ProductPhoto { get; set; }
        public string User { get; set; }
        
        
    }
}

File: client/src/_models/shoppingCart.ts

import {User} from '../_models/user'
export interface ShoppingCart{
    id?:number;
    productId:number;
    productName:string;
    productPric:number;
    selectedSize: string;   
    quantity:number;
    productPhoto:string;
    totalPrice:number;
    user:User;
}